mutation_partition.cc

```diff
@@ -680,7 +680,7 @@ void appending_hash<row>::operator()(Hasher& h, const row& cells, const schema&
     for (auto id : columns) {
         const cell_and_hash* cell_and_hash = cells.find_cell_and_hash(id);
         if (!cell_and_hash) {
-            return;
+            continue;
```

Comment:
Perhaps even simpler, we can return a "NULL" digest constant (?)  so the digest will be distinct.

Reply:
I'll rebase this pull request,  add a test case for `1,NULL,2` vs `1,2,NULL` and a digest constant. When I'm done, I guess I'll close this pull request and open a new one, since I don't have push rights to github.com/pdziepak/scylla.git.